### ISSUE TITLE
NVDA does not read chrome browser controls when hovering mouse over them.

### ISSUE THREAD

**Issue by NayanaLakhwani** [nvaccess/nvda#13843]
**Steps to reproduce:**
where I work I'm helping a user who needs to use chrome browser.
The user switches between using the mouse and the keyboard.
I'm totally blind so maybe details are missing.
Open Google chrome browser, now hover over the chrome browser interface mainly
toolbar where you can click on bookmarks,customize and control Google Chrome ellipsis, 
forward backward, refresh buttons and more.
Also hover over other areas of google chrome browser.

Actual behavior:
nvda is silent, not reading the chrome browser controls.

Expected behavior:
when hovering over mouse the chrome browser nvda should read what's under the mouse, just like it does in other programs.
Navigation using the keyboard is normal.

System configuration
NVDA installed/portable/running from source:
portable

NVDA version:
nvda.exe, NVDA 2022.1 & NVDA 2022.2.2 Beta versions

Windows version:
Windows 10 21H2 (OS Build 19044.1706)

Name and version of other software in use when reproducing the issue:
Google chrome browser 103.0.5060.53 (Official Build) (64-bit)
 & 102.0.5005.115 (Official Build) (64-bit).

Other information about your system:
Other questions
Does the issue still occur after restarting your computer?
yes

Have you tried any other versions of NVDA? If so, please report their behaviors.
no

If NVDA add-ons are disabled, is your problem still occurring?
yes

Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes

**Comment by seanbudd** [comment]
Related #11845

**Comment by feerrenrut** [comment]
Closing as a duplicate of https://github.com/nvaccess/nvda/issues/11845
